pkg/operation/botanist/component/fluentoperator/crds/crds.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Why are these CRDs in their own golang package? I don't think we do this for other CRDs like HVPA etc.